_posts/2019-04-03-live-view-with-pub-sub.md

```diff
@@ -0,0 +1,265 @@
+---
+author: Sophie DeBenedetto
+author_link: https://github.com/sophiedebenedetto
+categories: post
+date: 2019-04-03
+layout: post
+title:  Building Real-Time Features with Phoenix Live View and PubSub
+excerpt: >
+  Integrate Phoenix PubSub with LiveView to build real-time features capable of broadcasting updates across a set of clients.
+---
+
+In an [earlier post](https://elixirschool.com/blog/phoenix-live-view/), we used the brand new (still pre-release at time of writing) Phoenix LiveView library to build a real-time feature with very little backend code and even less JavaScript. LiveView allowed us to easily connect our client to the server via a socket and push updates down to our client. In an app that allows users to "deploy" a repo to GitHub, we achieved the following real-time functionality:
+
+<div class="responsive-embed">
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/8M-Hjj7IBu8" frameborder="0" allowfullscreen></iframe>
```

Comment:
Will these render correctly? They don't on GH.

Reply:
Yep, this is how we're rendering the video embeds for the pervious LiveView post